lib/app/ctx.js

```diff
@@ -220,10 +220,16 @@ Box.prototype.unwrap = function (data, content) {
             if (msg.type.substr(3) === '_CRYPT' || msg.type.substr(3) === '_SIGN') {
                 data = msg.contents;
             }
-
```

Comment:
I just made spacing consistent within a function. No problem to leave it as is, but I think the space there is not needed.